lib/legacy/index.ts

```diff
@@ -1,4 +1,5 @@
-import * as pluginInterface from './plugin';
-import * as monitorInterface from './monitor';
+import * as plugin from './plugin';
+import * as monitor from './monitor';
+import * as common from './monitor';
 
-export {pluginInterface, monitorInterface};
```

Comment:
TBH, this is technically breaking change. Also, I don't understand how naming change helps with `the need to reference dist`.

Reply:
we don't want to reference dist, so we want to structure our exports in a way we want to import them. Given how we use `dist` it seems we want to `types.legacy.plugin.Blah`
without it we will need to go `types.legacy. pluginInterface.Blah` agreed it needs a chat :)

Reply:
@kyegupov @gjvis @snyk/boost

Reply:
With this, I still cannot `import {Blah} from '@snyk/cli-interface/types/legacy/plugin'`.